condition_resource.go

```diff
@@ -0,0 +1,28 @@
+package ladon
+
+import (
+	"strings"
+)
+
+// ResourceCondition is a condition which is fulfilled if the request's subject is equal to the given value string
+type ResourceCondition struct{}
```

Comment:
This goes more in the direction of "the context must match a substring within the resource name" right?

Reply:
yes!

Reply:
Do you have an idea how the name of the struct could represent this?

Reply:
ResourceContainsCondition?
ResourceStringContainsCondition?